subworkflows/local/utils_nfcore_mcmicro_pipeline/main.nf

```diff
@@ -250,11 +249,18 @@ def expandSampleRow( row ) {
 // Generate methods description for MultiQC
 //
 def toolCitationText() {
-    // TODO nf-core: Optionally add in-text citation tools to this list.
     // Can use ternary operators to dynamically construct based conditions, e.g. params["run_xyz"] ? "Tool (Foo et al. 2023)" : "",
     // Uncomment function in methodsDescriptionText to render in MultiQC report
+    // TODO: not sure of the correct syntax for the ternary operators below.
```

Comment:
Syntax looks fine. Did you test to verify that the expected text shows up wherever it's supposed to in the output files?

Reply:
That's great about the syntax, thanks!
I thought this was something that would only run when we deployed, but you're right, I can install and run it locally.  So far its report is just showing module version numbers.  Looking into it more.

Reply:
Ah, I see.  We're not in the list of supported modules and the report only displays data for supported modules, so we're only seeing module version information.  If our modules were supported we'd see citation information at the beginning of the section for that module in the report.  This can be seen in [this example report](https://seqera.io/examples/bs-seq/multiqc_report).
I'll see what we need to do to have our modules supported.

Reply:
Let's hold off on the multiqc tickets since our tools don't yet have much of anything that multiqc could pick out and include in a report. We'll do that a bit later.